Licensing_Program/utils/load_config.py

```diff
@@ -54,7 +54,7 @@ def validate_config_stream(stream):
     return valid_config
 
 
-def load_config(proj_dir):
+def load_config(proj_dir="."):
```

Comment:
Not sure I understand what you're saying here. It sounds like you're arguing for removing the `proj_dir` argument entirely, since it probably won't be anything other than current directory.

My point was that, in terms of this utility function, assuming `"."` as a default makes more sense in software_dmv.py than in here.